Tasks/PublishTestResultsV1/publishtestresults.ts

```diff
@@ -20,7 +20,6 @@ tl.debug('publishRunAttachments: ' + publishRunAttachments);
 let matchingTestResultsFiles = ffl.findFiles(testResultsFiles, false, tl.getVariable('System.DefaultWorkingDirectory'));
 if(!matchingTestResultsFiles || matchingTestResultsFiles.length == 0) {
   tl.warning('No test result files matching ' + testResultsFiles + ' were found.');  
-  tl.exit(0);
```

Comment:
dont we have exit call in 'azure-pipelines-task-lib' ?

Reply:
Yes, It will not have a definition for exit

Reply:
I don't see any difference, I have verified the 2(before change and after change) runs of pipeline and there is no change in the execution of **Publish test results V1 Task**